data-pipeline/src/pipeline/rag.py

```diff
@@ -116,7 +116,7 @@ def category_stats(urn, category_name, data, ofsted_rating, rag_mapping, close_c
     key = "outstanding" if ofsted_rating.lower() == "outstanding" else "other"
     key += "_10" if close_count > 10 else ""
 
-    series = data[category_name]
+    series = data.loc[(data[category_name] > 0) | (data.index == urn), category_name]
```

Comment:
suggestion: move this to a distinct function.

I know it's only a line but given what we've been discussing elsewhere, I'd be tempted to pull this into a separate function, something like:

```python
def _get_series_for_category(data: pd.DataFrame, category: str, urn: int) -> pd.Series:
    """
    Some explanation of why we're doing this.
    """
    ...
```

A comment might suffice but it's not so much the _what_ that's complicated, rather the _why_.